src/peft/tuners/lora/config.py

```diff
@@ -357,6 +396,10 @@ def __post_init__(self):
                 raise ImportError("The required package 'scipy' is not installed. Please install it to continue.")
             if self.loftq_config is None:
                 raise ValueError("`loftq_config` must be specified when `init_lora_weights` is 'loftq'.")
+            
+        elif self.init_lora_weights == "eva":
+            if self.eva_config is None:
+                raise ValueError("`eva_config` must be specified when `init_lora_weights` is 'eva'.")
```

Comment:
Let's also check the reverse, i.e. if `self.init_lora_weights != "eva"`, there should be no `eva_config`.